obs_status.yml

```diff
@@ -1445,7 +1446,7 @@ obs:
   status: 1
 - agasc_id:
   - 27919752
-  comments: Mag 9.3 spoiler 26 arcsec away
+  comments: Mag 9.3 spoiler 26 arcsec away.  Set bad with bad position.
```

Comment:
strictly speaking, if this is bad position, then it is not a spoiler, so we could remove that, right?